cross_compilation/setup.sh

```diff
@@ -0,0 +1,18 @@
+set -e
+
+PLATFORM_DIR=$(realpath $(dirname $0)/..)
+
+mkdir -p $PLATFORM_DIR/cross_compilation/build
+cd $PLATFORM_DIR/cross_compilation/build
+
+# Cloning cross is necessary in order to get access to the `build-docker-image` command.
+git clone https://github.com/cross-rs/cross
+cd cross
+git submodule update --init --remote
```

Comment:
This doesn't make sense - why would cloning a repo suddenly add a cargo subcommand? Can you link me to what you read that said this

Reply:
Covered in pair programming session
https://github.com/cross-rs/cross-toolchains